Documentation/GettingStartedWithTheMRTK.md

```diff
@@ -12,157 +11,97 @@ To get started with the Mixed Reality Toolkit you will need:
+[![HandInteractionExample scene](../External/ReadMeImages/MRTK_Examples.png)](README_HandInteractionExamples.md)
 
-We have done an incredible amount of work to ensure the new project experience is as streamlined as possible.  To this end, we endeavoured to get you up and running as fast as possible.
+The [hand interaction examples scene](README_HandInteractionExamples.md) is a great place to get started with because it shows a wide variety of UX controls and interactions in MRTK. To get started we will import MRTK, open the example scene, and explore the scene in the editor.
+1. Create a new Unity project. We used 2018.3.9f1 for this tutorial, though any Unity 2018.3.X release should work.
 
-To get your first project up and running, the steps are as follows:
+2. Import the Microsoft.MixedRealityToolkit.Unity.Foundation package you've downloaded by going into "Asset -> Import Package -> Custom Package", selecting the .unitypackage file, ensuring all items to import are checked, and then selecting "Import".
 
-### 1. Create your new project (or start a new scene in your current project)
+3. Import Microsoft.MixedRealityToolkit.Unity.Examples.unitypackage following the same steps as above.
 
-> *Note* when creating a new project with Unity 2018, Unity gives you several templates to choose from.  Currently the **MRTK does not yet support the Scriptable Render Pipeline**, so the LWSRP, HDSRP and VRSRP projects are not compatible with MRTK projects.  Please stay tuned to the MRTK GitHub site for future announcements on SRP support.
+6. Open the HandInteractionExamples scene under `Assets\MixedRealityToolkit.Examples\Demos\HandTracking\Scenes\HandInteractionExamples`
 
-### 2. [Import the Mixed Reality Toolkit asset](https://github.com/Microsoft/MixedRealityToolkit-Unity/releases)
+7. You will get a prompt asking you to import "TMP Essentials". 
 
-> The Mixed Reality Toolkit is available via [multiple delivery mechanisms](DownloadingTheMRTK.md) and in the future will also be available via the Unity package manager once Unity makes that option available.
+![TMP Essentials](../External/Documentation/Images/MRTK_GettingStarted_TMPro.png)
 
-Get the asset and import it in to your Unity project using  "Asset -> Import Package -> Custom Package" from the Unity Editor menu.
+8. Select "Import TMP essentials" button. "TMP Essentials" refers to TextMeshPro plugin, which some of the MRTK examples use for improved text rendering.
 
-![](../External/ReadMeImages/Unity_ImportAssetOption.png)
+9. Close the TMPPro dialog. After this you need to reload Unity, so close unity and re-open unity, then open your project and go back to the scene.
 
-Accept all the content and continue.
+10. Finally, press the play button to start the scene and explore the scene! You can use simulated hands to interact in editor. Press WASD keys to fly around the scene. Press and hold right mouse to look around. Press and hold space bar to use a simulated hand.
```

Comment:
>start the scene and explore the scen [](start = 38, length = 36)

nit: scene appears twice within a span of 5 words, perhaps "start and explore the scene"?